app/src/main/java/cm/aptoide/pt/install/InstallAppSizeValidator.kt

```diff
@@ -20,4 +25,15 @@ class InstallAppSizeValidator {
     }
   }
 
+
+  private fun isAppAlreadyDownloaded(download: Download): Boolean {
+    for (fileToDownload in download.filesToDownload) {
+      if (!FileUtils.fileExists(
+              filePathProvider.getFilePathFromFileType(fileToDownload) + fileToDownload.fileName)) {
+        return false
+      }
+    }
+    return true
```

Comment:
isn't returning true as a default here dangerous? if you would pass and empty filesToDownload list it would say true, maybe add a check on it at the beggining?